server.js

```diff
@@ -31,6 +31,35 @@ app.get('/events/:id', (request, response) => {
   response.json(event);
 })
 
+app.delete('/events/:id', (request, response) => {
```

Comment:
Thanks for adding it. It will help us in the future and being able to remove positions right now too!